MyCryptoHQ/MyCrypto
# ENS MVP (by May 4th)

### Notes

- Branched from Michael's #2499 which fixes another little ENS issue
- More internal notes https://mycrypto.atlassian.net/wiki/spaces/IN/pages/180453542/ENS+Must+do+by+May+4th

### Description

- Sends people to ENS manager instead of legacy.mycrypto.com

### To Do Still

- [ ] Add the new / update translation strings back to ko.js, ja.js, fr.js
- [ ] Figure out which data currently displaying on current ENS implementation will be out of date.
- [ ] Remove or adjust label to clarify what the data is showing.   


### Changes

- Sends people to ENS manager instead of legacy.mycrypto.com on the following views:
    - AUCTION
    - FORBIDDEN
    - NOT YET AVAILABLE
    - OPEN
    - OWNED
    - REVEAL
- Changes the ensV3Url from `https://legacy.mycrypto.com/?ensname=${name}#ens` to `https://manager.ens.domains/name/${name}`

### Steps to Test

AUCTION
- I don't know how to make this show up

FORBIDDEN
- I don't know how to make this show up

NOT YET AVAILABLE
- I don't know how to make this show up

OPEN
- Search a long, random string of characters
- Check to ensure when you click the link, it takes you to the ENS manager in a new tab
- Check the name it takes you to in this new tab matches what user entered

OWNED
- Search mycrypto, myetherwallet, whatever
- Check to ensure when you click the link, it takes you to the ENS manager in a new tab
- Check the name it takes you to in this new tab matches what user entered

REVEAL
- search fullrefund. Or, look on https://twitter.com/EnsBot for **auctions** that were just started
- Check to ensure when you click the link, it takes you to the ENS manager in a new tab
- Check the name it takes you to in this new tab matches what user entered


### Screenshots

<img width="939" alt=" 2019-04-16 at 7 17 07 PM" src="https://user-images.githubusercontent.com/7924827/56256124-3e5cef80-607c-11e9-8b0b-90c17e555ec8.png">
<img width="933" alt=" 2019-04-16 at 7 16 53 PM" src="https://user-images.githubusercontent.com/7924827/56256125-3e5cef80-607c-11e9-8786-43b78b1e2773.png">
<img width="939" alt=" 2019-04-16 at 7 16 39 PM" src="https://user-images.githubusercontent.com/7924827/56256126-3e5cef80-607c-11e9-9b8a-8c0e07e1f211.png">
